behave/features/steps/general.py

```diff
@@ -67,3 +68,12 @@ def set_field_value_step(context, field_name, value):
     elem = context.browser.find_element_by_name(field_name)
     elem.send_keys(value)
     elem.submit()
+
+
+@when('I give the "{css_selector}" field the value "{element_value}" appended with todays date')
+def set_element_value_appended_with_today_date(context, css_selector, element_value):
+    html_element = context.browser.find_element_by_css_selector(css_selector)
```

Comment:
you can just invoke the `set_element_value` function here with your `element_value` variable

this amend will avoid any duplicated code in this pull request

Reply:
and done !